frontend/app/templates/components/account-form.hbs

```diff
@@ -4,68 +4,76 @@
   </Modal.header>
```

Comment:
Why was this file even touched here? Looks like that Bootstrap stuff... Is this here on purpose?

Reply:
Merge conflict.